readme/email_to_note.md

```diff
@@ -0,0 +1,27 @@
+# Email to note
+
+You can save your emails in Joplin Cloud by forwarding your emails to your Inbox Email address. The content of the email and its attachments will be saved in a note with the name of the email inside a notebook called Inbox. 
+
+All emails forwarded to your Inbox Email will use your Joplin Storage capacity.
+
+This feature is available for Pro and Teams members.
+
+## How to get your Inbox Email
+
+### Desktop
+
+To copy your Inbox Email you will need to navigate to the Options screen and then to the Joplin Cloud tab. You will need to have your synchronization set to Joplin Cloud.
+
+<img src="https://raw.githubusercontent.com/pedr/joplin/email-to-note-docs/Assets/WebsiteAssets/images/email_to_note/desktop.png" width="80%"/>
+
+### Mobile
+
+To copy your Inbox Email you will need to navigate to the Configurations screen and find the Joplin Cloud section. You will need to have your synchronization set to Joplin Cloud.
```

Comment:
When we mention the configuration screen, we link to it because there are different ways to access it depending on the device: `[Configuration screen](https://github.com/laurent22/joplin/blob/dev/readme/config_screen.md)`